### ISSUE TITLE
Network error while generating pdf in windows.

### ISSUE THREAD

**Issue by zealfire** [KnpLabs/snappy#165]
Hi thanks for the great wrapper. I am facing a problem while generating pdf in windows. The pdf is generated as expected but a run time exception is always thrown. The exception is as follows:

`RuntimeException: The exit status code '1' says something went wrong: stderr: "Loading pages (1/6) [> ] 0% [======> ] 10% [===========> ] 19% [============> ] 20% [================> ] 27% [==================> ] 30% [============================================================] 100% Counting pages (2/6) [============================================================] Object 1 of 1 Resolving links (4/6) [============================================================] Object 1 of 1 Loading headers and footers (5/6) Printing pages (6/6) [> ] Preparing [============================================================] Page 1 of 1 Done Exit with code 1 due to network error: ContentOperationNotPermittedError " stdout: "" command: C://xampp/htdocs/modules/pdf/wkhtmltopdf/bin/wkhtmltopdf.exe --lowquality "C:\Users\sony\AppData\Local\Temp\knp_snappy5590f02e7424a4.53154017.html" "queue.pdf". in Knp\Snappy\AbstractGenerator->checkProcessStatus()`

Can you help me to get rid of this problem? Thanks.

**Comment by jeremyb** [comment]
Hi,

Your `ContentOperationNotPermittedError` error seems to be related to wkhtmltopdf, After some researches, I think it can be related to:

* an invalid path (https://github.com/wkhtmltopdf/wkhtmltopdf/issues/1937)
* an obsolete option (https://github.com/wkhtmltopdf/wkhtmltopdf/issues/1877#issuecomment-69301913)

I hope it could help...
If not, can you provide more details about what options you're using and perhaps the `C:\Users\sony\AppData\Local\Temp\knp_snappy5590f02e7424a4.53154017.html` content.

**Comment by zealfire** [comment]
Hi,
   Thanks for looking into the problem. On further investigation I came to conclusion that inclusion of base tag in head section was cause of problem. I was writing a code like this:

`<base href=" \path\to\file" \>`
which was causing  `Exit with code 1 due to network error: ContentOperationNotPermittedError` but after removing it the error was removed. Any reasons for this?

Since I think this is something caused by wkhtmltopdf so I have made an issue regarding the same over here: https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2448

**Comment by akovalyov** [comment]
@zealfire do you have any update on it?
can I close it as not related to Snapy itself?

**Comment by akovalyov** [comment]
btw, have you tried ```--load-error-handling ignore```?

**Comment by akovalyov** [comment]
@zealfire any news on it?

**Comment by zealfire** [comment]
@akovalyov  sorry for replying so late. I think that this problem is not because of snappy hence earlier  I made an issue regarding this in the wkhtmltopdf repo. But since then I have heard from them. Also recently I changed my HTML content after which I am not getting any error hence I didn't paid attention to this issue in the past few days. Any ways since this problem is not related to this library I am closing this issue. Thanks for showing patience.
